### PR TITLE
feat: adding WORKSPACE_NAMESPACE substitution var

### PR DESCRIPTION
**Pull request by cwyl02**
for https://jira.d2iq.com/browse/D2IQ-79902 and https://jira.d2iq.com/browse/D2IQ-81196

### CODE REVIEW

**Review by gracedo — APPROVED** [review]

**Review by cbuto — APPROVED** [review]

File: helm-repositories/banzaicloud/banzaicloud.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ apiVersion: source.toolkit.fluxcd.io/v1beta1
 kind: HelmRepository
 metadata:
   name: kubernetes-charts.banzaicloud.com
+  namespace: ${WORKSPACE_NAMESPACE}
```
Comment (cbuto) [inline helm-repositories/banzaicloud/banzaicloud.yaml:6]: should this be `${workspaceNamespace}`?

Comment (gracedo) [reply helm-repositories/banzaicloud/banzaicloud.yaml:6]: oops you're right, it should be `${workspaceNamespace}`

**Review by gracedo — CHANGES_REQUESTED** [review]
sorry @cbuto 's right, should be `${workspaceNamespace}`

File: helm-repositories/banzaicloud/banzaicloud.yaml (hunk 1 above)
Comment (cwyl02) [reply helm-repositories/banzaicloud/banzaicloud.yaml]: Done

**Review by cbuto — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Pull request merged**